src/Chip/Chip.js

```diff
@@ -0,0 +1,338 @@
+
+    /**
+     * Override the inline-styles of the chip.
+     */
+    chipStyle: React.PropTypes.object,
+
+    /**
+     * CSS `className` of the root element.
+     */
+    className: React.PropTypes.node,
+
+    /**
+     * If true, the Chip displays a delete icon.
+     */
+    deletable: React.PropTypes.bool,
+
+    /**
+     * Can be used to render a label in the Chip.
+     */
+    label: React.PropTypes.node,
+
+    /**
+     * Override the label color.
+     */
+    labelColor: React.PropTypes.string,
+
+    /**
+     * Override the inline-styles of the label.
+     */
+    labelStyle: React.PropTypes.object,
+
+    /**
+     * Callback function fired when the element loses focus.
+     * @param {object} event `blur` event targeting the element.
+     */
+    onBlur: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the element gains focus.
+     * @param {object} event `focus` event targeting the element.
+     */
+    onFocus: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the element is focused or blurred by the keyboard.
+     *
+     * @param {object} event `focus` or `blur` event targeting the element.
+     * @param {boolean} keyboardFocused Indicates whether the element is focused.
+     */
+    onKeyboardFocus: React.PropTypes.func,
+
+    /**
+     * Callback function fired when a mouse button is pressed down on the element.
+     *
+     * @param {object} event `mousedown` event targeting the element.
+     */
+    onMouseDown: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the mouse enters the element.
+     *
+     * @param {object} event `mouseenter` event targeting the element.
+     */
+    onMouseEnter: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the mouse leaves the element.
+     *
+     * @param {object} event `mouseleave` event targeting the element.
+     */
+    onMouseLeave: React.PropTypes.func,
+
+    /**
+     * Callback function fired when a mouse button is released on the element.
+     *
+     * @param {object} event `mouseup` event targeting the element.
+     */
+    onMouseUp: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the close button is clicked.
+     * @param {object} event `touchTap` event targeting the element.
+     */
+    onRequestClose: React.PropTypes.func,
+
+    /**
+     * Callback function for when a touchTap event ends.
+     */
+    onTouchEnd: React.PropTypes.func,
+
+    /**
+     * Callback function for when a touchTap event starts.
+     */
+    onTouchStart: React.PropTypes.func,
+
+    /**
+     * Callback function fired when the `Chip` element is touch-tapped.
+     *
+     * @param {object} event TouchTap event targeting the `Chip` element.
+     */
+    onTouchTap: React.PropTypes.func,
+
+    /**
+     * Override the inline-styles of the root element.
+     */
+    style: React.PropTypes.object,
+  };
+
+  static defaultProps = {
+    onBlur: () => {},
```

Comment:
I'm wondering what the best choice between using a default function and checking that the function property exist.